htdocs/user/card.php

```diff
@@ -1562,6 +1586,30 @@
 			print '</td>';
 			print "</tr>\n";
 
+			// Expense report validator
+			print '<tr><td>'.$langs->trans("UserExpenseValidator").'</td>';
+			print '<td>';
+			if (empty($object->fk_user_expense_validator)) print $langs->trans("None");
```

Comment:
There is already a rule that say that the default user that validate is the supervisor.
So this field can be used to overwrite this rule, so if fk_user_expense_validator is empty, it does not means there is nobody to validate but that the validator is the User supervisor (it chief). So i would replace the
print $langs->trans("None");
with 
print $langs->trans("Default");

Reply:
I get your point. I'd rather keep a simple translation such as "Expense report validator" and add a "?" next to it to explain that if no validator is defined, the manager is used instead. What do you think about it ?